### DISCUSSION TITLE
How to enable the build animation?

### DISCUSSION THREAD

**Discussion by LeifHenriksen** [kurkle/chartjs-chart-treemap#57] [Q&A]
Hi,

I noticed that when using version 1.0.2 that there is no building animation (the animation that plays when the graph spawns), in this code pen you can see it, if you change **chartjs** and **chartjs-chart-treemap** to the old version you get the animation back. https://codepen.io/leifhenriksen/pen/porxqar

The chart in the docs also has no animation : https://chartjs-chart-treemap.pages.dev/usage.html

So my question is, how do I enable the build animation?

Thank you

**Comment by kurkle** [comment]
I don't recall what bug this was, but this has been fixed in later versions.
